sdk/eventhub/Azure.Messaging.EventHubs.Processor/src/EventProcessorClient.cs

```diff
@@ -365,6 +365,9 @@ public new bool IsRunning
         ///
         internal MessagingClientDiagnostics ClientDiagnostics { get; }
 
+        /// <inheritdoc />
+        protected override bool? IsBatchTracingEnabled { get => false; }
```

Comment:
Do you mind moving L360 (CheckpointStore) down after this property?   I try to keep them organized by visibility and it looks like one snuck in out-of-order.